src/quacc/recipes/espresso/phonons.py

```diff
@@ -84,6 +90,201 @@ def phonon_job(
     )
 
 
+@job
+def q2r_job(
+    prev_dir: SourceDirectory,
+    fildyn: str = "matdyn",
+    parallel_info: dict[str] | None = None,
+    **calc_kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a basic q2r.x calculation. It should allow you to
+    use all the features of the [q2r.x binary](https://www.quantum-espresso.org/Doc/INPUT_Q2R.html#idm51)
+
+    This job requires the results of a previous ph.x calculation, you might
+    want to create your own flow to run both jobs in sequence.
+
+    Parameters
+    ----------
+    prev_dir
+        Outdir of the previously ran ph.x calculation. This is used to copy
+        the the dynamical matrix files.
+    fildyn
```

Comment:
I'd prefer something a bit more descriptive like `matdyn_file` (unless `fildyn` is widely acknowledged by  the Espresso community).